.github/workflows/ci-cd.yml

```diff
@@ -74,9 +74,9 @@ jobs:
 
       - name: analyze ${{ matrix.lib }}
```

Comment:
This file should remain unchanged as the source for the actions is still rotorsoft org. Anyways, we need to properly fix the pipeline to work on andela-technology org.